nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/controller-services/controller-services.effects.ts

```diff
@@ -382,9 +383,9 @@ export class ControllerServicesEffects {
                     const serviceId: string = request.id;
 
                     const enableDialogReference = this.dialog.open(EnableControllerService, {
+                        ...XL_DIALOG,
                         data: request,
-                        id: serviceId,
-                        panelClass: 'large-dialog'
+                        id: serviceId
```

Comment:
Yes this was intentional in order to keep the text for each of the 'Steps to Disable' on a single row with the green check mark at the end of the single line.

Reply:
Should we be using the `basis-2/3`. and `basis-1/3` here to make the left side larger than the right? Seems like we probably don't need half of the space for the referencing components here.

Reply:
This dialog still doesn't look good. The removal of additional height does not work. The referencing component tree can be very big. I think we need to allocate enough (I realize this is subjective) height so accommodate this listing. The area is designed so scroll for when the listing gets really big. If we're going to 2/3 and 1/3 I think we probably want to stick with XL since the referencing component is a tree component which can be nested deeply. If we want to stick with L maybe 1/2 and 1/2 is more appropriate.

This comment applies to any place we are showing referencing components.

<img width="774" alt="Screenshot 2024-03-29 at 11 33 10 AM" src="https://github.com/apache/nifi/assets/123395/4f789f20-9cf6-4a95-8457-84634c330650">